scripts/tvm_cli/tvm_cli.py

```diff
@@ -25,10 +25,12 @@
 import tensorflow as tf
 import numpy as np
 
+OUTPUT_PREPROCESSING_MODULE_FILENAME = "preprocess.so"
 OUTPUT_NETWORK_MODULE_FILENAME = "deploy_lib.so"
 OUTPUT_NETWORK_GRAPH_FILENAME = "deploy_graph.json"
 OUTPUT_NETWORK_PARAM_FILENAME = "deploy_param.params"
 OUTPUT_CONFIG_FILENAME = "inference_engine_tvm_config.hpp"
+OUTPUT_PREPROCESSING_CONFIG_FILENAME = "preprocessing_inference_engine_tvm_config.hpp"
```

Comment:
We don't really need another header file. The reason I'm doing this is because:
1. If we put `preprocessing` into `inference_engine_tvm_config.hpp`, more information is needed. For example, the input and output of `scatter` and `centerpoint_backbone` are actually different. We'll have to add something like this to the template. 
```
  {
  {%- for node in input_list %}
    {"{{ node['name'] }}", {{ node['dtype_code'] }}, {{ node['dtype_bits'] }}, {{ node['lanes'] }}, {{ '{' }}{{ node['shape']|join(', ') }}{{ '}}' }}{{ ',' if not loop.last }}
  {%- endfor %}
  },  // preprocessing_inputs

  {
  {%- for node in output_list %}
    {"{{ node['name'] }}", {{ node['dtype_code'] }}, {{ node['dtype_bits'] }}, {{ node['lanes'] }}, {{ '{' }}{{ node['shape']|join(', ') }}{{ '}}' }}{{ ',' if not loop.last }}
  {%- endfor %}
  }  // preprocessing_outputs
```
But we don't implement preprocessing/postprocessing using tvm most of time. At least for now, it only makes sense for multistage inference because we don't want to too much data transfer between devices. 

2. Minimum amount of changes of code. :)

Reply:
Oh, right. I was looking at the resulting header files and those were the only differences. In that case please double check which information gets written in the input and output of the preprocessing header file, because at the moment it is the same as the main one.

Reply:
fixed